content-src/lib/snippets.js

```diff
@@ -388,7 +388,7 @@ export function addSnippetsSubscriber(store) {
     // snippets.initialized:                  Is SnippetsProvider currently initialised?
     if (state.Prefs.values["feeds.snippets"] &&
       // If the message center experiment is enabled, don't show snippets
-      !state.Prefs.values.asrouterExperimentEnabled &&
+      (!state.Prefs.values.asrouterExperimentEnabled && state.Prefs.values.asrouterOnboardingCohort === 0) &&
```

Comment:
To make this less confusing I would create a variable up top that's like `isASRouterEnabled`and used that instead